src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +29,13 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
+        //List<Product> products = null; //imitation no connection with DB
         // TODO: add log message about successful fetched data from DB
+        if (products != null) {
+            logger.debug("Data from the DB received successfully: Product = {}", products);
+        } else {
+            logger.error("Data from the DB not received");
+        }
```

Comment:
I dont think we need to add if here, lets just log info about products, if List is null we will see it from log